.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+# Trigger this workflow on release
+on:
+  release:
+    types: [published]
```

Comment:
I think we should build the standalone version on every `master` push and on release tags, like for the winbuld release:

https://github.com/wanadev/yoga/blob/master/.github/workflows/winbuild.yml#L3-L9

Else we might discover that something is broken at the time of the release.